### PR TITLE
Update German translation, some minor English fixes, update language files

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/german.txt [hunk 1]
```diff
@@ -1284,10 +1275,61 @@ Regular Background Color
...
+== NA
+
+SA
+== SA
+
+CHN
+== CHN
+
+Getting server list from master server
+== Serverliste von Masterserver holen
```
Comment (heinrich5991) [inline data/languages/german.txt:1308]: Rather: "Hole Serverliste vom Masterserver"? It's displayed in the center of the server browser while the list is being requested.

File: data/languages/german.txt [hunk 2]
```diff
@@ -1284,10 +1275,61 @@ Regular Background Color
...
+
+Leak IP
+== IP durchsickern lassen
```
Comment (heinrich5991) [inline data/languages/german.txt:1311]: I have a feeling that this is too long, I think the space was pretty tight.

**Review by heinrich5991 — COMMENTED** [review]
LGTM otherwise.

**Comment by def-** [comment]
Done

**Comment by heinrich5991** [comment]
bors r+

Thanks.

**Pull request merged**